content/blog/2025/02/2025-02-05-command-palette.adoc

```diff
@@ -0,0 +1,33 @@
+---
+layout: post
+title: "A new way to search in Jenkins 2.492.1"
+tags:
+- jenkins
+- user-experience
+authors:
+- janfaracik
+opengraph:
+  image: /images/post-images/2025/02/2025-02-05-command-palette/opengraph.png
+discourse: true
+---
+
+image::/images/post-images/2025/02/2025-02-05-command-palette/opengraph.png[role=center]
+
+We’re happy to announce that link:https://jenkins.io/changelog-stable/2.492.1[Jenkins 2.492.1] comes with a
```

Comment:
```suggestion
We’re happy to announce that link:/changelog-stable/2.492.1[Jenkins 2.492.1] comes with a
```
Suggestion to adjust link to direct link in site.

Reply:
Good idea, will update to `/changelog/2.492.1` if https://github.com/jenkins-infra/jenkins.io/pull/7838 gets merged.